modules/swagger-codegen/src/main/resources/nodejs/controller.mustache

```diff
@@ -9,12 +9,23 @@ module.exports.{{nickname}} = function {{nickname}} (req, res, next) {
   {{#allParams}}
   var {{paramName}} = req.swagger.params['{{baseName}}'].value;
   {{/allParams}}
+  var responseHeaders = [];
+  var header = {};
+  var responseHeaderSchema;
+  {{#responses}}
+  {{#headers}}
+  responseHeaderSchema = {{{jsonSchema}}};
```

Comment:
Is this approach a good idea if multiple headers are defined or is iterating through "headers" object better?